### ISSUE TITLE
List Indices must be integers or slices, not str

### ISSUE THREAD

**Issue by Kzcclabs** [h2oai/h2ogpt#652]
![Error](https://github.com/h2oai/h2ogpt/assets/106029474/c5779585-9584-4a7c-88a8-de618e1e75ec)

I have downloaded the Windows CPU version of H2O.ai GPT but when I go to Models tab, select llama base model, then click load to download from preset URL. I get the attached error. What can I do to fix this

**Comment by pseudotensor** [comment]
I tried the CPU installer just now and for those same steps I hit no error.  I also made a query and it had no issue.

DId you do any other steps besides what you listed, such as different model URL or something?

You can help diagnose by running instead this line in the windows command prompt (not miniconda or anything, just normal windows command line):

```
C:\Users\pseud\AppData\Local\Programs\h2oGPT\Python\python.exe C:\Users\pseud\AppData\Local\Programs\h2oGPT\h2oGPT.launch.pyw"
```

That is, normally the command the h2oGPT icon runs uses "pythonw.exe" and hides the output, but if you run with "python.exe" it'll still work but show in the console the output.

Then when you hit the error, you should see the full stack trace not visible in UI.

Thanks!

**Comment by Kzcclabs** [comment]
Thanks for getting back to me so quickly. I tried the command on the cmd and there is no error. The issue I am having is that I have no model so the search wont work. When I go to the models tab and select the base model as llama and click on Load-Unload Model/LORA, I hit the List Indices must be integers or slices, not str error.

However I downloaded the "llama-2-7b-chat.ggmlv3.q8_0.bin" file and placed in the env folder of my repo and then tried again, now I do not get that error but the processing goes on and on and I left it and went to sleep and when I woke up I had a local host time out so I restarted my PC and trying it out again. Any idea if I am in the right direction or wrong direction. I need to install the model to get it to work :(. I have been trying for 2 daays now

![image](https://github.com/h2oai/h2ogpt/assets/106029474/7dc1d1f8-1905-4d15-be5b-a6a7ef57d12c)
